pitop/camera/core/cameras/usb_camera.py

```diff
@@ -41,14 +47,19 @@ def __del__(self):
 
     def get_frame(self):
         # Always PIL format
-        return Image.frombytes(
+        pil_image = Image.frombytes(
             'RGB',
             (self.__camera.width, self.__camera.height),
             self.__camera.get_frame(),
             'raw',
             'RGB'
         )
 
+        if self._rotate_angle != 0:
+            pil_image = pil_image.rotate(angle=self._rotate_angle, expand=True)
+
+        return pil_image
+
```

Comment:
```suggestion
        return Image.frombytes(
            'RGB',
            (self.__camera.width, self.__camera.height),
            self.__camera.get_frame(),
            'raw',
            'RGB'
        ).rotate(angle=self._rotate_angle, expand=True)

```

Reply:
Should be smart enough to just do nothing if angle is 0, so no need for storing to a variable